src/Components/Carousel/Carousel.js

```diff
@@ -2,14 +2,27 @@ import React, {useEffect, useState} from 'react';
 import './Carousel.scss';
 
 import useFetch from '../useFetch/useFetch';
+import Modal from '../Modal/Modal'
```

Comment:
We can end statements with or without `;` . Good practice is to be consistent. Decide how you want to write statements and stick to it.